components/src/core/Drawer/DrawerNavGroup.tsx

```diff
@@ -291,10 +296,19 @@ function NavigationListItem(
     const statusColor = (navItem as NavItem).statusColor;
 
     return (
-        <div style={{ position: 'relative' }} className={`${classes.listItem} ${active && classes.listItemNoHover}`}>
+        <div
+            style={{ position: 'relative' }}
+            className={clsx(
+                defaultClasses.listItemContainer,
+                classes.listItemContainer,
+                active && defaultClasses.listItemNoHover
+            )}
+        >
             {active && (
                 <div
-                    className={`${classes.active} ${activeBackgroundShape === 'square' ? classes.square : ''}`}
+                    className={`${defaultClasses.active} ${
+                        activeBackgroundShape === 'square' ? defaultClasses.square : ''
+                    }`}
```

Comment:
you can use clsx to conditionally apply a class.
Also, do we want to allow customization here?

Reply:
I added a customization to the active class, I left the square class uncustomizable because they can opt to not sure it.

Reply:
Used clsx to make the class concatentation look a lot prettier